osgtest/library/core.py

```diff
@@ -134,6 +140,13 @@ def remove_log():
     """Removes the detailed log file; not for general use."""
     os.remove(_log_filename)
 
+def get_stat(filename):
+    '''Return stat for 'filename', None if the file does not exist'''
+    try:
+        return os.stat(filename)
+    except OSError, exc:
+        if exc.errno == errno.ENOENT:
```

Comment:
OK, so if the file does not exist, you return None. What if there is a different error? Then you still return None, IIRC?